opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/__init__.py

```diff
@@ -196,24 +213,52 @@ def create_observable_counter(
             self._instrument_id_instrument[instrument_id] = instrument
             return instrument
 
-    def create_histogram(self, name, unit="", description="") -> APIHistogram:
+    def create_histogram(
+        self,
+        name: str,
+        unit: str = "",
+        description: str = "",
+        advisory: Optional[MetricsHistogramAdvisory] = None,
+    ) -> APIHistogram:
+        if advisory is not None:
+            invalid_advisory = False
+            try:
+                boundaries = advisory.explicit_bucket_boundaries
+                invalid_advisory = not (
+                    boundaries
+                    and all(isinstance(e, float) for e in boundaries)
+                )
+            except (KeyError, TypeError):
+                invalid_advisory = True
+
+            if invalid_advisory:
+                advisory = None
+                _logger.warning(
+                    "Advisory must be a dict with explicit_bucket_boundaries key containing a sequence of numbers"
```

Comment:
Think you need to update the log message now it's not a dict

Reply:
Good catch